docs/data_sources/other_data.rst

```diff
@@ -31,24 +31,20 @@ The `original EPA CAMD to EIA crosswalk <https://github.com/USEPA/camd-eia-cross
 was published by the US Environmental Protection Agency on GitHub and connects EPA CAMD
 emissions units (smokestacks) which appear in :doc:`epacems` with corresponding EIA
 plant components reported in EIA Forms 860 and 923 (``plant_id_eia``, ``boiler_id``,
-``generator_id``). This one-to-many connection is necessary because pollutants from
+``generator_id``). This many-to-many connection is necessary because pollutants from
 various plant parts are collecitvely emitted and measured from one point-source.
 
 The original crosswalk was generated using only 2018 data. However, there is useful
 information in all years of data, and we augment the crosswalk that they publish on
-GitHub by running their code against 2018 and all later years of data.
+GitHub by running their code against all available later years of data.
 
 Re-running the crosswalk pulls the latest data from the
 `CAMD FACT API <https://www.epa.gov/power-sector/field-audit-checklist-tool-fact-api>`__
 which results in some changes to the generator and unit IDs reported on the EPA side of
-the crosswalk.  The changes only result in the addition of new units and generators in
+the crosswalk. The changes only result in the addition of new units and generators in
 the EPA data, with no changes to matches at the plant level (other than identification
-of new plant-plant matches). However, the updates to generator and unit IDs have
-resulted in changes to the subplant IDs - some EIA boilers and generators which
-previously had no matches to EPA data have now been matched to EPA unit data, resulting
-in an overall reduction in the number of rows in the
-:ref:`core_epa__assn_eia_epacamd_subplant_ids` table. See :issue:`4039` and :pr:`4056`
-for a discussion of the changes observed when we made these changes.
+of new plant-plant matches). Note that the sub-plant IDs are not necessarily stable
+across multiple releases of this data, and should not be hard-coded into analyses.
```

Comment:
We derive sub-plant IDs (``subplant_id``) from the crosswalk in the table :ref:`core_epa__assn_eia_epacamd_subplant_ids`. Note that these IDs are not necessarily stable across multiple releases of this data, and should not be hard-coded into analyses.

Reply:
added!